frontend/src/views/Storage/StorageModal.tsx

```diff
@@ -55,6 +55,9 @@ export default function StorageModal({
     if (formData.manureType === undefined) throw new Error('Form validation failed.');
     if (annualPrecipitation === undefined) throw new Error('No precipitation data found.');
 
+    if (!formData.uuid) {
```

Comment:
Why would there not be a uuid here?

Reply:
A new storage system, inheriting the default data, would not have a UUID.